tensorflow/compiler/mlir/tensorflow/tests/tf_trait_folds.mlir

```diff
@@ -143,4 +143,194 @@ func @testTripleLogicalNot(%arg0: tensor<i1>) -> tensor<i1> {
   %2 = "tf.LogicalNot"(%1) : (tensor<i1>) -> tensor<i1>
   // CHECK: return [[LNOT]]
   return %2: tensor<i1>
-}
\ No newline at end of file
+}
+
+// CHECK-LABEL: func @testSingleAbs
+// CHECK-SAME:  ([[ARG0:%.+]]: tensor<i8>)
+func @testSingleAbs(%arg0: tensor<i8>) -> tensor<i8> {
+  // CHECK: [[ABS:%.+]] = "tf.Abs"([[ARG0]])
+  %0 = "tf.Abs"(%arg0) : (tensor<i8>) -> tensor<i8>
+  // CHECK: return [[ABS]]
+  return %0: tensor<i8>
+}
+
+// CHECK-LABEL: func @testDoubleAbs
+// CHECK-SAME:  ([[ARG0:%.+]]: tensor<i8>)
+func @testDoubleAbs(%arg0: tensor<i8>) -> tensor<i8> {
+  // CHECK: [[ABS:%.+]] = "tf.Abs"([[ARG0]])
+  %0 = "tf.Abs"(%arg0) : (tensor<i8>) -> tensor<i8>
+  %1 = "tf.Abs"(%0) : (tensor<i8>) -> tensor<i8>
+  // CHECK: return [[ABS]]
+  return %1: tensor<i8>
+}
+
+// CHECK-LABEL: func @testSingleCeil
+// CHECK-SAME:  ([[ARG0:%.+]]: tensor<f1>)
```

Comment:
`f1` is not a defined type (and is failing this test). Did you mean to use `f32`?